presentation/src/main/java/com/kdjj/presentation/di/PresentationModule.kt

```diff
@@ -20,4 +21,9 @@ class PresentationModule {
             RingtoneManager.getDefaultUri(RingtoneManager.TYPE_NOTIFICATION)
         )
     }
-}
\ No newline at end of file
+
+    @Provides
+    fun provideWorkManager(
+        @ApplicationContext context: Context
+    ) = WorkManager.getInstance(context.applicationContext)
```

Comment:
명확하게 적고 싶으시면 변수명을 applicationContext로 하는게 어떨까요?
```suggestion
        @ApplicationContext applicationContext: Context
    ) = WorkManager.getInstance(applicationContext)
```